js/src/behaviors/segmented-controls.ts

```diff
@@ -0,0 +1,50 @@
+import { TabContainerElement, TabContainerChangeEvent } from '@github/tab-container-element';
+
+export function initSegmentedControls() {
+	initSegmentedControlChangeListener();
+	initSegmentedControlNavClickListener();
+}
+
+function initSegmentedControlNavClickListener() {
```

Comment:
I am actually in favor (I think) of using `tab-container` from github. But I don't understand why we'd need to augment it in this way. The tab container's description notes that it already handles accessibility. What is it not handling that we must update it?

And if we do need to change it, perhaps we should consider whether it might be better to create our own, similar version of this component.

Reply:
@wibjorn This augmentation is specifically for adding the "previous"/"next" buttons, which `tab-container` does not support out of the box.

Reply:
Best that we augment this in a most web-component centric way then I think. Perhaps we need to create our own element by extending the tab element, something very basically like this:

```ts
class SegmentedTabs extends TabContainerElement {
   constructor(..args) {
     super(..args);
   }
}

// in definition file

const root = (typeof globalThis !== 'undefined' ? globalThis : window) as typeof window
try {
  root.SegmentedTabs = SegmentedTabs.define()
} catch (e: unknown) {
  if (
    !(root.DOMException && e instanceof DOMException && e.name === 'NotSupportedError') &&
    !(e instanceof ReferenceError)
  ) {
    throw e
  }
}
```

Don't think we want to mix and match elements/behaviors. Two different patterns that don't need to be melded.

Another option is copy the source into Atlas JS and change it directly to meet our needs. 
https://github.com/github/tab-container-element/blob/main/src/index.ts

Reply:
Which option is favored here?

Reply:
I would favor the former for now, can always do the latter later if we need to. The element is still maintained, so I don't see much reason to copy all that over.

Reply:
I went ahead with the whole copy of the source and add our logic (next/prev) to the class.  The above option was not playing as nice as I would have liked.  

@wibjorn  Please let me know your thoughts on the latest commits as soon as you can.